zsh-abbr.zsh

```diff
@@ -929,16 +929,16 @@ _zsh_abbr_job_push() {
     job_group=${(q)2}
     timeout_age=30 # seconds
 
-    job_dir=${TMPDIR:-/tmp}zsh-abbr-jobs/${job_group}
+    job_dir=${TMPDIR:-/tmp}/zsh-abbr-jobs/${job_group}
```

Comment:
macOS's `$TMPDIR` includes a trailing slash. In all cases the new `/` needs to be inside this braces, so `${TMPDIR:-/tmp/}`.